tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py

```diff
@@ -12,3 +12,6 @@ try:
 except time_interface.TimeError:
   # That's what we expect
   pass
+else:
+  # we have to fail this if we try didnt raise a exception for not registering time method
+  log("[FAIL]: The call to time_interface.time_updatetime() method has succeeded but instead it should have raised time_interface.TimeError exception.  This exception has to be raised if time_register_method() which registers the (tcp / ntp) method is not called prior to calling time_interface.time_updatetime() method \n")
```

Comment:
Apart from this string that wraps over multiple lines, this looks mergeable (once you fix the long string :).

Test results:
$ python utf.py -f ut_seattlelib_time_interface_updatetime_before_registermethod.r2py
Testing module: seattlelib
	Running: ut_seattlelib_time_interface_updatetime_before_registermethod.r2py [ PASS ]
$ vi ut_seattlelib_time_interface_updatetime_before_registermethod.r2py